### ISSUE TITLE
[Assistive tech and device support] Alert and status messages aren't announced without receiving focus. (11.22.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#39847]
#### VFS team name
Cedar-1095b

#### VFS product name
1095B Tax Form

#### VFS feature name
1095B Tax Form

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Alert and status messages aren't announced without receiving focus.

### Issue Details
Since I'm reviewing based on wireframes and can't currently test with the live authenticated experience, I'm logging this as a prospective issue -- one that hopefully has already been considered and doesn't actually need to be resolved! I'd like to understand what happens after a user clicks the download button. How long does it take for the PDF to be generated and downloaded? During that time, is there any kind of on-screen progress or status indicator? And finally, after download, where does keyboard focus go?

### VA.gov Experience Standard
[Category Number 11, Issue Number 22](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 4.1.3_AA

---

### Platform Recommendation
Depending on how long the PDF generation/download takes, an ARIA `role="status"` region may be needed to communicate that something is happening after pressing the button. After the action is complete (success or failure), focus should be set programmatically to either the "Download complete" heading or the "Sorry, something went wrong" heading using `tabindex="-1"`. This might be something you've already accounted for! If not, I would love to have a follow-up conversation prior to launch and/or test it immediately after launch.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
